apps/tensor_times_vector/tensor_times_vector.cpp

```diff
@@ -1,26 +1,43 @@
 #include <iostream>
+#include <chrono>
 #include "taco.h"
 
+using namespace std::chrono;
 using namespace taco;
 
 int main(int argc, char* argv[]) {
-  Format csr({Dense,Sparse});
-  Format csf({Sparse,Sparse,Sparse});
+  // Format csr({Dense,Sparse});
```

Comment:
instead of modifying the application here, it would better to add a few tests that utilize galloping so that they run in CI.